### PR TITLE
MSMQ configuration

### PR DESCRIPTION
**Pull request by gbiellem**
Relates to https://github.com/Particular/docs.particular.net/issues/917

### CODE REVIEW

**Comment by gbiellem** [comment]
@SimonCropp  is this what you had in mind in #917

File: nservicebus/msmq/index.md [hunk 1]
```diff
@@ -8,11 +8,44 @@ redirects:
...
 MSMQ is the default transport used by NServiceBus.
 
-
 ## NServiceBus Configuration
 
-NServiceBus requires a specific MSMQ configuration to run. To enable this configuration either use "NServiceBus Prerequisites" in the [Platform Installer](/platform/installer/) or using the [Powershell Module](/nservicebus/operations/management-using-powershell.md).
+NServiceBus requires a specific MSMQ configuration to operate.     
+
+The supported configuration is to only have the base service installed with no optional features.
+The MSMQ items listed within the Add/Remove Windows Features and the associated DISM.exe command line are not the same between version.   To enable the supported configuration either use "NServiceBus Prerequisites" in the [Platform Installer](/platform/installer/) or using the [Powershell Module](/nservicebus/operations/management-using-powershell.md).
```
Comment (SimonCropp) [inline nservicebus/msmq/index.md]: "same between versions."

**Comment by SimonCropp** [comment]
@gbiellem it is. is there something not finished? ie u still have a WIP on it

**Comment by gbiellem** [comment]
No its complete. I left in WIP as i wasn' t sure If I'd addressed what you
wanted in the PR. It was ages ago you asked me to do it and we never
captured the scope in the issue.

**Pull request merged**